tags/resources/tag-formation-templates/template-roles.md

```diff
@@ -1,3 +1,103 @@
 # Roles within the `<TAG NAME>`
 
-*tbd* - contributions are welcome!
+The TAG includes several key roles that are critical to the group's success.
+Beyond the roles described in this document the TAG has many contributors, all serving in varying capacities.
+
+- [Roles within the `<TAG NAME>`](#roles-within-the-tag-name)
+  - [Chair](#chair)
+  - [Technical lead](#technical-lead)
+  - [Working Group Lead](#working-group-lead)
+  - [Project Lead](#project-lead)
+  - [TOC liaison](#toc-liaison)
+  - [Emeritus Chair](#emeritus-chair)
+
+## Chair
+
+A Chair drives the CNCF community efforts of the TAGs domain.
+Chairs are experienced leaders within the CNCF community with rooted expertise in the TAG area. Chairs serve a two-year term but may renew their term by submitting a follow-up application.
+Chair applications go through an [election process](https://github.com/cncf/toc/blob/master/tags/cncf-tags.md#elections) with a final CNCF TOC vote.
+The [leadership election process is described here](template-leadership-election-process.md#chair).
+
+* Manage the operations and governance of the group.
+* Organize, host, plan and facilitate TAG meetings and events.
+* Reporting to the CNCF TOC on the status of TAG work.
+* Encouraging community members to start projects and working groups.
+* Resolve technical difficulties and decisions related to multiple sub-projects.
+* Remind the community of the scope of the TAG and point out guardrails in discussions, working groups, and projects.
+* Serve as TAG leadership representative in CNCF project discussions and at community events.
+* Evolve the TAG to reflect ongoing changes in the industry.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and promote diversity in TAG work.
+* Enforce and promote good communication in TAG efforts in accordance with the [CNCF CoC](https://www.cncf.io/conduct/)
```

Comment:
```suggestion
The TAG Chairs drive, guide, and coordinate the TAG's efforts within the broader CNCF ecosystem and within their TAG's community. They are supported by Technical Leads in the execution of TAG activities
Chairs may be experienced leaders within the CNCF ecosystem and often have specialized domain expertise. Chairs have term limits, most commonly two years. Chairs continue to serve until removed either by an elected replacement or by the TOC.

Chair applications go through an [election process](https://github.com/cncf/toc/blob/master/tags/cncf-tags.md#elections) with a final CNCF TOC vote.

The [leadership election process is described here](template-leadership-election-process.md#chair).

Chair responsibilities and expectations may include the following or delegation of the following: 
* Manage the operations and governance of the group.
* Organize, host, plan and facilitate TAG meetings and events.
* Reporting to the CNCF TOC on the status of TAG work.
* Encouraging community members to start projects and working groups.
* Resolve technical difficulties and decisions related to multiple sub-projects.
* Remind the community of the scope of the TAG and point out guardrails in discussions, working groups, and projects.
* Serve as TAG leadership representative in CNCF project discussions and at community events.
* Evolve the TAG to reflect ongoing changes in the industry.
* Onboard and mentor new community members.
* Mentor community members in a leadership role within the TAG.
* Enforce and promote diversity in TAG work.
* Enforce and promote good communication in TAG efforts in accordance with the [CNCF CoC](https://www.cncf.io/conduct/)
* Embody and promote excellent leadership in accordance with the [Technical Leadership principles](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#technical-leadership-principles)

For more information on TAG Chair responsibilities and expectations, please refer the to the TOC's description of [TAG Chairs](placeholder)
```

Reply:
@erinaboyd - Leo pulled together an initial list pertinent to the specific TAG operations, I'm sure there are elements here you've already captured in your draft for Chair responsibilities. I've added a placeholder here on line 38 for when we've got them drafted in a PR. I'd think keeping our defined responsibilities and expectations separate from the template here (for use by TAGs) is reasonable. What are your thoughts?

Reply:
Perfect. I will add the proposed suggestion already, but keep this discussion open, so we don't lose track, to update the placeholder, and add possible other suggestions.

Reply:
@erinaboyd, if you find a bit of time, could you take at this section? Thanks! :)

> @erinaboyd - Leo pulled together an initial list pertinent to the specific TAG operations, I'm sure there are elements here you've already captured in your draft for Chair responsibilities. I've added a placeholder here on line 38 for when we've got them drafted in a PR. I'd think keeping our defined responsibilities and expectations separate from the template here (for use by TAGs) is reasonable. What are your thoughts?